src/main/java/com/minecolonies/coremod/client/gui/WindowHutAllInventory.java

```diff
@@ -147,7 +147,9 @@ private void updateResources()
 
                 for (final Map.Entry<ItemStorage, Integer> entry : storage.entrySet())
                 {
-                    items.add(new ItemStorage(entry.getKey().getItemStack(), entry.getValue(), false).getItemStack());
+                	ItemStack stack = entry.getKey().getItemStack().copy();
```

Comment:
This does not work officially, and even though it works in the current Mc version it is not officially supported. As such please change this to use itemstorage instances or use the proposed change by @Raycoms.

Reply:
should i just change the items list to use ItemStorage instead of ItemStack then?

Reply:
I did get rid of it now.